hack/install-etcd.sh

```diff
@@ -21,7 +21,40 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+LOG_DIR=${LOG_DIR:-"/tmp"}
+
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::etcd::install
+if [ $# -lt 1 ] ; then
+    kube::etcd::install
+else
+  echo "The current operation is $1"
+  case $1 in
+    add)
+      if [ -n "$2" ]  && [ -n "$3" ]; then
+        kube::etcd::add_member $2 $3
+      fi
+      ;;
+    start)
+      # install etcd if necessary
+      if ! [[ $(which etcd) ]]; then
+        if ! [ -f "${KUBE_ROOT}/third_party/etcd/etcd" ]; then
```

Comment:
Sometimes etcd is installed in different location. Perhaps need to check whether etcd is in path or not